### DISCUSSION TITLE
The Week 7... (Minimal Update)

### DISCUSSION THREAD

**Discussion by ghost** [luckydog7/Funkin-android#1442] [mods]
Info (OG):#1356
Info (COLOR):#1357

:link:(OG)https://cntr.click/92gJBy7
:link:(COLOR)https://cntr.click/H2r1t7W
:link:Mods:https://cntr.click/34Dwpz8
:link:(Bruh 7):https://cntr.click/JZt1bXN
(Week 7 Mod)
Used Mode + Video:https://youtu.be/WGJ6MBP92RU
:link:(Brugh Windows Port):https://cntr.click/5rWy4D1
[Not Tested]

versions:Custom Week (4.0)

Credits
Prime
Akki
Cowhat
(SHOUTOUT)
FridayNightSans
NeonSus
Iwasgt
Cronix

enjoy!

**Comment by Jotaro-Gaming** [comment]
can you make a new link for the og one?
Thanks!
